src/components/DiagramFrame/SeqDiagram/MessageLayer/MessageLabel.vue

```diff
@@ -3,8 +3,8 @@
     title="Double click to edit"
     class="px-1 cursor-text right hover:text-skin-message-hover hover:bg-skin-message-hover"
     :class="{
-      'py-1 px-2 ml-1 cursor-text': editing,
```

Comment:
It adds unwanted padding that makes the editing box jump.
<img width="561" alt="image" src="https://github.com/mermaid-js/zenuml-core/assets/169417/96bd75b6-67b0-4c3f-98dd-52cf04ddfda7">

Reply:
I aimed to create more space for the editor box. Should this adjustment cause any instability, feel free to eliminate the styles entirely.